static/src/js/components/FormFields/AccessMethodRadio/__tests__/AccessMethodRadio.test.js

```diff
@@ -81,17 +87,99 @@ describe('AccessMethodRadio component', () => {
     expect(screen.getByText('Something went wrong')).toBeInTheDocument()
   })
 
-  test('displays the details as a tooltip', async () => {
-    setup()
-    const user = userEvent.setup()
-    const icon = screen.getByTestId('edsc-icon-details')
-    await waitFor(async () => {
-      await user.hover(icon)
+  describe('tool-tip on the access method radio', () => {
+    test('displays the details as a tooltip on hover', async () => {
+      setup()
+      const user = userEvent.setup()
```

Comment:
https://testing-library.com/docs/user-event/intro#writing-tests-with-userevent

```We recommend invoking [userEvent.setup()](https://testing-library.com/docs/user-event/setup) before the component is rendered.```

I know this was copied from an existing test, but we found that recommendation in the Testing Library docs, so when you see `userEvent.setup()` after the `render()` call we should update it. 

The way we have been doing it is at the beginning of `setup` call `const user = userEvent.setup()`, and return that user as part of the returned object in the setup method